### PR TITLE
Adding chaos exporter to agent manifest for chaos events and metrics

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan@chaosnative.com>

## Proposed changes

Adding chaos exporter to agent manifest for chaos events and metrics

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Special notes for your reviewer:
PR for 2.0 beta 1

### CODE REVIEW

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml [hunk 1]
```diff
@@ -165,6 +165,8 @@ spec:
               value: "litmuschaos/chaos-operator:1.13.0"
             - name: LITMUS_CHAOS_RUNNER_IMAGE
               value: "litmuschaos/chaos-runner:1.13.0"
+            - name: LITMUS-CHAOS-EXPORTER_IMAGE
+              value: "litmuschaos/chaos-exporter:v1.13.x"
```
Comment (rajdas98) [inline litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: Is it the image tag- v1.13.x

Comment (ishangupta-ds) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: yes @rajdas98

Comment (rajdas98) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: I don't think we create this kind of image tag.

Comment (ishangupta-ds) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: I have been using this, here is another PR for the same, https://github.com/litmuschaos/litmus/pull/2438 cc: @ispeakc0de @ksatchit

File: litmus-portal/graphql-server/manifests/namespace/ 2b_litmus_deployment.yaml [hunk 2]
```diff
@@ -52,4 +52,30 @@ spec:
...
             - name: OPERATOR_NAME
-              value: "chaos-operator"
\ No newline at end of file
+              value: "chaos-operator"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: chaos-monitor
```
Comment (rajdas98) [inline litmus-portal/graphql-server/manifests/namespace/ 2b_litmus_deployment.yaml:61]: better to name it as chaos-exporter

Comment (rajdas98) [reply litmus-portal/graphql-server/manifests/namespace/ 2b_litmus_deployment.yaml:61]: it will create confusion later, better to refactor right now.

File: litmus-portal/cluster-k8s-manifest.yml [hunk 3]
```diff
@@ -169,6 +169,8 @@ spec:
               value: "litmuschaos/chaos-operator:1.13.0"
             - name: LITMUS_CHAOS_RUNNER_IMAGE
               value: "litmuschaos/chaos-runner:1.13.0"
+            - name: LITMUS-CHAOS-EXPORTER_IMAGE
```
Comment (rajdas98) [inline litmus-portal/cluster-k8s-manifest.yml:172]:
```suggestion
            - name: LITMUS_CHAOS_EXPORTER_IMAGE
```

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: That does not work, i have tried using that tag @rajdas98

File: litmus-portal/graphql-server/manifests/cluster/2b_litmus_rbac.yaml [hunk 4]
```diff
@@ -70,4 +70,21 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: litmus-cluster-scope
-    namespace: #{AGENT-NAMESPACE}
\ No newline at end of file
+    namespace: #{AGENT-NAMESPACE}
+---
```
Comment (rajdas98) [inline litmus-portal/graphql-server/manifests/cluster/2b_litmus_rbac.yaml:74]: k8s service shouldn't be present inside RBAC file

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: 1.13.0 is present in dockerhub. https://hub.docker.com/r/litmuschaos/chaos-exporter/tags?page=1&ordering=last_updated

File: litmus-portal/graphql-server/manifests/cluster/2b_litmus_rbac.yaml (hunk 4 above)
Comment (rajdas98) [reply litmus-portal/graphql-server/manifests/cluster/2b_litmus_rbac.yaml:74]: no need to create, put it inside `2c_litmus_deployment.yaml`

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:169]: okay i am updating it, i tried `v1.13.0`

File: litmus-portal/graphql-server/graph/generated/generated.go [hunk 5]
```diff
@@ -3365,7 +3365,7 @@ input UpdateMyHub {
 	SSHPublicKey: String
 }
 `, BuiltIn: false},
-	&ast.Source{Name: "graph/project.graphqls", Input: `type Project {
```
Comment (rajdas98) [inline litmus-portal/graphql-server/graph/generated/generated.go:3368]: Are these changes required?

Comment (ishangupta-ds) [reply litmus-portal/graphql-server/graph/generated/generated.go:3368]: I just did a gofmt @rajdas98

**Review by rajdas98 — APPROVED** [review]
